tools/tertiary-analysis/scanpy/anndata_operations.xml

```diff
@@ -298,14 +314,24 @@ adata.write('output.h5', compression='gzip')
       <param name="color_by" value="louvain"/>
       <output name="output_h5ad" file="anndata_ops.h5" ftype="h5ad" compare="sim_size"/>
     </test>
+    <test>
+      <param name="input_obj_file" value="anndata_ops.h5"/>
+      <param name="from_var" value = "gene_symbols" />
+      <param name="to_var" value = "hello_all" />
+      <output name="output_h5ad" ftype="h5ad">
+        <assert_contents>
+          <has_h5_keys keys="var/hello_all" />
+        </assert_contents>
+      </output>
+    </test>
     <test>
       <param name="input_obj_file" value="find_cluster.h5"/>
       <param name="input_format" value="anndata"/>
       <conditional name="copy_r">
         <param name="default" value="true"/>
         <param name="r_source" value="read_10x.h5"/>
       </conditional>
-      <output name="output_h5ad" file="anndata_ops_raw.h5" ftype="h5ad" compare="sim_size">
+      <output name="output_h5ad" ftype="h5ad">
```

Comment:
```suggestion
      <output name="output_h5ad" file="anndata_ops_raw.h5" ftype="h5ad" compare="sim_size">
```

We do need the file size check for this other test, and the file is already there, so no point in removing the size check.